.github/workflows/lint-test.yaml

```diff
@@ -0,0 +1,55 @@
+# docs ref: https://github.com/helm/chart-testing-action
+name: Lint and Test Charts
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  lint-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.4.0
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.0.1
+
+      - name: Run chart-testing (list-changed)
+        id: list-changed
+        run: |
+          changed=$(ct list-changed --chart-dirs charts)
+          if [[ -n "$changed" ]]; then
+            echo "::set-output name=changed::true"
+          fi
+
+#      - name: Run chart-testing (lint)
+#        run: ct lint
+
+      - name: Create kind cluster
+        uses: helm/kind-action@v1.0.0
+        if: steps.list-changed.outputs.changed == 'true'
+        with:
+          node_image: kindest/node:v1.18.15
+
+      # don't run `ct install` if charts hasn't been changed
+      - name: Run chart-testing (install)
+        run: |
+          kubectl create ns vela-system
+          ct install --namespace vela-system
```

Comment:
It should have a `helm test` runing here to validate, I didn't see them?

https://helm.sh/docs/topics/chart_tests/

Reply:
`ct install` would run both helm `install` and `test`.

[Here](https://github.com/helm/chart-testing/blob/master/doc/ct_install.md#ct-install) is the doc ref.

Reply:
I see, thanks, but use a yaml about `Application` seems not check the check well.

Reply:
We should at least check the Application is running and has a deployment and ingress created

Reply:
I agree, an application just test whether crd is installed.

Will fix it.